examples/wandb_sweeps/make_wandb_sweep.py

```diff
@@ -45,36 +45,38 @@
 }
 
 
-def make_sweep(project: str) -> int:
+def make_sweep(project: str, sweep_name: str) -> int:
```

Comment:
why not call it `sweep` for compatibility with `project`?

Reply:
Was thinking to make it clear that this is not the `sweep_id`, however, since the `id` is generated by this script, I think your suggestion makes sense :).